scripts/generate/templates/SlotDerivation.js

```diff
@@ -77,7 +77,13 @@ const mapping = ({ type }) => `\
  */
 function deriveMapping(bytes32 slot, ${type} key) internal pure returns (bytes32 result) {
     assembly ("memory-safe") {
-        mstore(0x00, key)
+        ${
+          type == 'address'
+            ? 'mstore(0x00, shr(96, shl(96, key)))'
```

Comment:
In packing we clean by doing

```suggestion
            ? 'mstore(0x00, and(key, shl(96, not(0))))'
```

I'm thinking we should an helper for cleaning